client/network/src/protocol/message.rs

```diff
@@ -147,7 +144,7 @@ pub mod generic {
 		/// Block announce.
 		BlockAnnounce(BlockAnnounce<Header>),
 		/// Transactions.
-		Transactions(Transactions<Extrinsic>),
+		Transactions(sc_network_transactions::Transactions<Extrinsic>),
```

Comment:
I checked the code and I don't think this is used anymore? If yes, could we please keep the variant here in the enum, but don't reference `sc_network_transactions`